pkg/cloud/libvirt/actuators/machine/actuator.go

```diff
@@ -307,8 +307,8 @@ func (a *Actuator) applyMachineStatus(
 	now := metav1.Now()
 	machineCopy.Status.LastUpdated = &now
 	_, err = a.clusterClient.ClusterV1alpha1().
-		Machines(machineCopy.Namespace).Update(machineCopy)
-
+		Machines(machineCopy.Namespace).UpdateStatus(machineCopy)
+	glog.Infof("error in updating: %v", err)
```

Comment:
If there's no error we will print nothing meaningful which will be misleading. Also, if it's an error then `.Errorf`.